server/src/main/java/io/atomix/copycat/server/state/MemberState.java

```diff
@@ -214,7 +218,7 @@ MemberState setNextIndex(long nextIndex) {
    * @return Indicates whether an append request can be sent to the member.
    */
   boolean canAppend() {
-    return !appending;
+    return appending < MAX_APPENDS && System.currentTimeMillis() - (timeBuffer.average() / MAX_APPENDS) >= appendTime;
```

Comment:
Whether an `AppendRequest` can be sent to a follower depends on two factors:
* Less than `2` requests are already outstanding
* More than half the average round trip time for the last 8 non-empty requests has elapsed since the start of the last request